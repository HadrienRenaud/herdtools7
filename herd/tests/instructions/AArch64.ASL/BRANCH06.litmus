AArch64 BRANCH06
{
0:X0=1;
0:X1=1;
}

 P0           ;
CMP W1,#1     ;
B.EQ .+8      ;
ADD W0,W0,#1  ;
forall 0:X0=1
