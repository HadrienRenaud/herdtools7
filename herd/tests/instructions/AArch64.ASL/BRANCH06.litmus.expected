Test BRANCH06 Required
States 1
0:X0=1;
Ok
Witnesses
Positive: 1 Negative: 0
Condition forall (0:X0=1)
Observation BRANCH06 Always 1 0
Hash=30a651cabc68f6be659b5554416283de

